.travis.yml

```diff
@@ -10,6 +10,8 @@ deploy:
 env:
   global:
     - PORT=3000
-    - NODE_ENV=production
-    - JWT_KEY=secret
+    - NODE_ENV="developement"
+    - JWT_KEY="secret"
```

Comment:
Are there not any alternatives to committing your `JWT_KEY` to a public repository?

Reply:
Oh. I don't know if there are. But I have an Idea

Reply:
What if I encrypt it with bcrypt ? 
so that I will be decrypting it when I want to use it in my code?

Reply:
I don't think that's a good idea. Let it stay as it is.

Reply:
But do research and see if you can add environment variables on Travis outside code.

Reply:
Okay. I'm on it